### PR TITLE
update liveness timeout

### PR DESCRIPTION
**Pull request by shileifu**

### CODE REVIEW

**Review by kevincai — APPROVED** [review]

File: pkg/be_controller/be_pod.go [hunk 1]
```diff
@@ -177,8 +177,7 @@ func (be *BeController) buildPodTemplate(src *v1alpha12.StarRocksCluster, beconf
 			}}},
 		},
 		LivenessProbe: &corev1.Probe{
-			FailureThreshold: 60,
```
Comment (dengliu) [inline pkg/be_controller/be_pod.go:180]: I understand there is a default value for `FailureThreshold` .
however, would it be better to set it explicitly for better readability?

File: pkg/cn_controller/cn_pod.go [hunk 2]
```diff
@@ -150,8 +150,7 @@ func (cc *CnController) buildPodTemplate(src *v1alpha12.StarRocksCluster, cnconf
 			}}},
 		},
 		LivenessProbe: &corev1.Probe{
-			FailureThreshold: 60,
```
Comment (dengliu) [inline pkg/cn_controller/cn_pod.go:153]: ditto

File: pkg/fe_controller/fe_pod.go [hunk 3]
```diff
@@ -194,8 +194,7 @@ func (fc *FeController) buildPodTemplate(src *v1alpha12.StarRocksCluster, feconf
 			}}},
 		},
 		LivenessProbe: &corev1.Probe{
-			FailureThreshold: 60,
```
Comment (dengliu) [inline pkg/fe_controller/fe_pod.go:197]: ditto

File: pkg/be_controller/be_pod.go (hunk 1 above)
Comment (shileifu) [reply pkg/be_controller/be_pod.go:180]: OK

**Review by kevincai — APPROVED** [review]

**Review by dengliu — APPROVED** [review]

**Pull request merged**